.circleci/dist_compile.yml

```diff
@@ -31,6 +31,28 @@ commands:
             git submodule sync --recursive
             git submodule update --init --recursive
 
+  install-duckdb:
```

Comment:
This is good for now but in a follow up we should add this to the docker images.

Reply:
Agreed! I don't have the power to update docker images.

Reply:
Actually you have :) We have a workflow in place that updates the dockerfiles after changes to the files are merged to main: https://github.com/facebookincubator/velox/actions/workflows/docker.yml

Reply:
Ah! Good to know. Thanks.